### PR TITLE
[#1733] Ability to remove Facebook authentication

### PR DESCRIPTION
**Pull request by bianca93**
# Description

Just like with Google authentication, we want to allow users
to remove Facebook auth by setting a new password.

This PR replicates the UI we have for Google auth.

## Corresponding Issue

https://github.com/ifmeorg/ifme/issues/1733

# Screenshots
Screenshot 2020-05-06 at 20 02 29

---

Reviewing this pull request? Check out our [Code Review Practices](https://github.com/ifmeorg/ifme/wiki/Code-Review-Practices) guide if you haven't already!

### CODE REVIEW

**Comment by bianca93** [comment]
I would like to add screenshots and tests, but I haven't figured out how to test Facebook auth locally and also don't know if we have any UI tests apart from unit tests on components. @bmwachajr @julianguyen

**Comment by julianguyen** [comment]
We don't have any tests (acceptance tests should be enough) for the account settings page but I think it's something we should add if you have the bandwidth. We should at least add it for these changes :)

As for testing locally, you'll need to generate a developer API key. Here are some instructions I put together recently: https://github.com/ifmeorg/ifme/wiki/Installation#facebook-oauth-20-ids

**Comment by bmwachajr** [comment]
Oh just seen this, great work so far, How is it going @bianca93 ?

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -29,6 +29,12 @@
       <%= raw t('devise.registrations.google_auth1') %>
     </div>
   <% end %>
+  
+  <% if current_user.provider == 'facebook' %>
```
Comment (julianguyen) [inline app/views/devise/registrations/edit.html.erb:33]: It would be great to add an acceptance test here for this and Google as auth well.

**Comment by julianguyen** [comment]
I'm gonna merge this PR, but we should make a follow-up one to address acceptance testing!

**Pull request merged**